docs/deploy-development-rp.md

```diff
@@ -76,42 +76,109 @@
+
+| Address Space | Address Range         | Address Count |
+| ------------- | --------------------- | ------------- |
+| 10.0.0.0/24   | 10.0.0.0 - 10.0.0.255 | 256 |
+| 10.1.0.0/24   | 10.1.0.0 - 10.1.0.255 | 256 |
+
+1. Create a new "Virtual Network Gateway (Gateway type: VPN)" in the Azure Portal manually. This needs to be configured to the "Virtual Network" named "rp-vnet" which will already existing in the shared RP's resource group. This new VPN will allow the local ARO-RP to connect to the to the existing "rp-vnet" to create a cluster. 
+
+1. Configure the new "rp-vnet" VPN with the same public certificate used for the existing dev-vpn. This is done at: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/<subscription-id>/resourceGroups/<aro-shared-rp-rg>/providers/Microsoft.Network/virtualNetworkGateways/rp-vnet/pointtositeconfiguration. You can simply copy the info from: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/<subscription-id>/resourceGroups/<aro-shared-rp-rg>/providers/Microsoft.Network/virtualNetworkGateways/dev-vpn/pointtositeconfiguration.
+
+1. Connect to the "rp-vnet" VPN created above. You can use openvpn or the azure vpn client, both have worked fine in our testing. 
+  1. Go to Point-to-Site Configuration for "rp-vnet" (https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/<subscription-id>/resourceGroups/<aro-shared-rp-rg>/providers/Microsoft.Network/virtualNetworkGateways/rp-vnet/pointtositeconfiguration) and download the VPN Client Certificate to your local environment. You can extract the zip file anywhere you would like, but we put it under the "secrets" folder because that is where the ARO-RP secrets reside.
+  1. For openvpn:
+    1. Copy ./secrets/rp-vnet/OpenVPN/vpnconfig.json to ./secrets/vpn-rp-eastus.ovpn
+    1. Copy the last two certificates ("P2S client certificate" and "P2S client certificate private key") from ./secrets/vpn-eastus.ovpn file to ./secrets/vpn-rp-eastus.ovpn. You can overwrite the placeholders for those certificates at the bottom in the ./secrets/vpn-rp-eastus.ovpn file.
+    1. Execute openvpn secrets/vpn-rp-eastus.ovpn. You may need sudo depending on your environment.
+  1. For azure vpn client (the azure vpn client for windows appears to require extra efforts; this is only for MacOS atm):
+    1. Click the 'import' button in the vpn list, you will be prompted with an "open file dialog".
+    1. Select the file: ./secrets/rp-vnet/AzureVPN/azurevpnconfig.xml. The data will be filled into the import screen with the exception of "Client Certificate Public Key Data" and "Private Key".
+    1. Copy the "P2S client certificate" into the "Client Certificate Public Key Data" field and "P2S client certificate private key" into the "Private Key" field.
+    1. Click "Save" and you should see your newly created VPN connection in the VPN list on the left.
```

Comment:
The numbered steps don't appear to be parsing correctly - see https://github.com/Azure/ARO-RP/blob/62a4bb6ef2f981e7d3db66a80a54dcf79410858b/docs/deploy-development-rp.md#preparation-to-create-cluster

Reply:
Fixed.